addon/mixins/touch-action.js

```diff
@@ -38,7 +41,7 @@ export default Mixin.create({
   ignoreTouchAction: false,
 
   init() {
-    this._super();
+    this._super(...arguments);
```

Comment:
remove `...arguments` - this is a very hot code path - hat tip to @runspired for noticing this in a previous round where i did the same thing. U don't need it here, nor do u want it.

Reply:
@eriktrom why do we not want this?

Reply:
runspired told me to remove it when i updated this file last year - he said that `...arguments` slows things down(which is true, i later found out when writing an unrelated algo), and that its not needed either in this case (b/c its a mixin? that part im not sure 'why' but i trust him on it 👍)

Reply:
I haven't heard of spread arguments being a perf hit before.  Or is it just because of the babel transpilation?  Happy to revert this back however as it shouldn't make a difference.

Reply:
babel transpilation per this discussion (of this file) 2 years ago though, hazy memory - https://github.com/html-next/ember-hammertime/pull/27#discussion_r81058157, didn't quote it quite right, actually quoted this instead (a couple comments down) - https://github.com/html-next/ember-hammertime/pull/27#discussion_r81058654

(side note per my comment of my more recent encounter with passing `...arguments` to a helper fn, inside an O(n) loop, perf_hooks caught it, i have not dug deeper, it was pretty recent finding, passing named args manually fixed it)

hope that clarifies things - for both of us now haha